zookeeper/core/component.py

```diff
@@ -1,5 +1,4 @@
-"""
-Components are generic, modular classes designed to be easily configurable.
+"""Components are generic, modular classes designed to be easily configurable.
```

Comment:
I don't mind changing the docstring format, but can we do it in a different PR and apply it to the whole repo? It will make review of this one easier.